modules/alloy/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Changed
+- `ModalDialogApis.getFooter` now returns an `Optional<AlloyComponent>` instead of `AlloyComponent`. #TINY-9996
```

Comment:
that's a breaking change, needs a major

Reply:
@TheSpyder Should I revert the corresponding change in alloy then? `getFooter` would just be a `getOrDie` since now the footer component is optional.

Reply:
Ah wait I forgot this is just an Alloy change, I've incremented the version of alloy to 13.0.0 in versions.txt

Reply:
Apologies for not being clearer in my response